src/Autocomplete/src/Controller/EntityAutocompleteController.php

```diff
@@ -76,7 +76,11 @@ private function getExtraOptions(Request $request): array
             return [];
         }
 
-        $extraOptions = $this->getDecodedExtraOptions($request->query->getString(self::EXTRA_OPTIONS));
+        try {
+            $extraOptions = $this->getDecodedExtraOptions($request->query->getString(self::EXTRA_OPTIONS));
+        } catch (\JsonException) {
+            throw new BadRequestHttpException('The extra options cannot be parsed.');
+        }
```

Comment:
Maybe pass the JsonException as previous? Not sure if relevant.

Reply:
Done